app/controllers/users_controller.rb

```diff
@@ -22,8 +30,56 @@ def create
     end
   end
 
+  def edit
+    @user = User.find_by id: params[:id]
+  end
+
+  def update
+    if @user.update_attributes(user_params)
+      flash[:success] = t ".prupdate"
+      redirect_to @user
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    if @user && @user.destroy
```

Comment:
No need to check @user here as you are using filter load_user for the destroy action.